extensions/amp-accordion/1.0/component.js

```diff
@@ -507,3 +507,29 @@ export function AccordionContent({
     </WithAmpContext>
   );
 }
+
+/**
+ * @param {{current: ?}|undefined} ref
+ * @return {Element|undefined}
+ */
+function checkRef(ref) {
+  if (!ref) {
+    return ref;
+  }
+
+  // in amp mode, the imperative handle gives us an object where the element
+  // property refers to the actual DOM element i.e. {element: DOMElement}
+  // in preact mode, we get the actaul DOM element (more details in: #33499)
+  const element = ref.current;
+  if (!element) {
+    return element;
+  }
+
+  // preact mode
+  if (element instanceof Element) {
```

Comment:
This is going to have issues with cross-document boundaries.
```suggestion
  if (element.nodeType === 1) {
```

Reply:
This is no longer needed.